<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharGen</class>
 <widget class="QMainWindow" name="CharGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharGen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>460</height>
    </size>
   </property>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>460</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>460</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="standardMode">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>275</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Стандартный режим</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomMode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>275</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Случайный режим</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="nextStep">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>385</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
